diff --git a/node_modules/sequelize/types/lib/model.d.ts b/node_modules/sequelize/types/lib/model.d.ts
index 29a754e..63bb4db 100644
--- a/node_modules/sequelize/types/lib/model.d.ts
+++ b/node_modules/sequelize/types/lib/model.d.ts
@@ -1972,14 +1972,13 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
   /**
    * Builds a new model instance and calls save on it.
    */
-  public static create<
-    M extends Model,
-    O extends CreateOptions<M['_attributes']> = CreateOptions<M['_attributes']>
-  >(
-    this: ModelStatic<M>,
-    values?: M['_creationAttributes'],
-    options?: O
-  ): Promise<O extends { returning: false } | { ignoreDuplicates: true } ? void : M>;
+   public static create<
+   M extends Model>(
+   this: {new (): M},
+   values?: object,
+   options?: CreateOptions
+ ): Promise<M>;
+ public static create(values: Object, optiona: CreateOptions & {returning: false}) : Promise<void>;
 
   /**
    * Find a row that matches the query, or build (but don't save) the row if none is found.
